.github/renovate.json

```
@@ -19,11 +19,11 @@
         "kroxylicious-kms-provider-hashicorp-vault-test-support/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/VaultTestKmsFacade.java",
         "kroxylicious-kms-provider-hashicorp-vault/src/test/java/io/kroxylicious/kms/provider/hashicorp/vault/TestVault.java",
         "kroxylicious-systemtests/src/main/resources/helm_vault_overrides.yaml",
-        "performance-tests/docker-compose.yaml"
+        "performance-tests/perf-tests.sh"
       ],
       "matchStrings": [
         "DockerImageName\\.parse\\(\"(?<depName>hashicorp/vault):(?<currentValue>\\d+\\.\\d+.\\d+)\"\\)",
-        "image:.*(?<depName>hashicorp/vault):(?<currentValue>\\d+\\.\\d+.\\d+)",
+        "\"${DOCKER_REGISTRY}/(?<depName>hashicorp/vault):(?<currentValue>\\d+\\.\\d+.\\d+)",
```

You might need to escape the $ sign, as this is significant to regex.

However you can probably omit it the DOCKER_REGISTRY part.  `(?<depName>hashicorp/vault)` etc should match the new string.  

```suggestion
        "(?<depName>hashicorp/vault):(?<currentValue>\\d+\\.\\d+.\\d+)",
```

---

Let me check that, sounds good

---

Yes, it works. Thanks!